dcpy/lifecycle/distribute/README.md

```diff
@@ -0,0 +1,57 @@
+# Distribute
+
+## Terms
```

Comment:
it'd be helpful to add `Dataset Package` to the list of terms, maybe last

Reply:
@damonmcc for sure. I'm curious your thoughts regarding `Dataset` vs `Dataset Package`. It sort of feels like Dataset is an abstract concept or standard, whereas the Package is the concrete file instantiation. Is that too heady? 😂

Reply:
agreed that they feel distinct. to me, Dataset is a concept and Package is a set of files

a Package could have multiple Datasets in it right? so a Product has one or more Packages. each Package has one or more Datasets

Reply:
Good question... I think there are probably two package concepts, the `product package` which would contain potentially multiple `dataset package`s. So for something like PLUTO you'd have a folder for e.g. 24v2 (the `product package`) which would contain a `dataset package` for the main files, and another `dataset package` for the changefile dataset

Reply:
that makes sense. now I feel like there are two qualities of a Package (either product or dataset):
- a collection of files
- everything a distribution endpoint needs for an update

Reply:
If we leaning towards using "dataset package", I agree on adding Dataset Package to Terms... For example, reading the red portion below makes me check the Terms section:

<img width="859" alt="image" src="https://github.com/NYCPlanning/data-engineering/assets/144725249/b60510ae-2aee-4f77-89da-9399c4ef9030">

Otherwise, I'm personally indifferent about dataset vs dataset package as long as it's explicitly defined.